pkg/printers/internalversion/describe_test.go

```diff
@@ -94,6 +94,30 @@ func TestDescribePodNode(t *testing.T) {
 	}
 }
 
+func TestDescribePodNodeNone(t *testing.T) {
+	fake := fake.NewSimpleClientset(&api.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "bar",
+			Namespace: "foo",
+		},
+		Spec: api.PodSpec{
+			NodeName: "",
+		},
+		Status: api.PodStatus{
+			HostIP: "",
+		},
+	})
+	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
+	d := PodDescriber{c}
+	out, err := d.Describe("foo", "bar", printers.DescriberSettings{ShowEvents: true})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if !strings.Contains(out, "<none>") {
```

Comment:
This seems inadequate. Wouldn't any such pods also have `Tolerations: <none>` ?  Seems you need to look for `Node: <none>`